### PR TITLE
Circleci

### PR DESCRIPTION
**Pull request by fuentesloic**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+machine:
+  node:
+    version: v5.4.1
```
Comment (vinyll) [inline circle.yml]: That's a very old version. Now it's beyond 10!
Can't you just say `stable`?

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    - node_modules
+
+test:
+  override:
+    - npm run lint
+    
+deployment:
+  release:
+    commands:
+      - npm run dev
```
Comment (vinyll) [inline circle.yml]: Is this because you want the styleguide for github pages?

Comment (fuentesloic) [reply circle.yml]: Our css I write as postcss with nested, ect ect (not supported by native css). So I need the builded css file
As we don't want dev push built file, circleCI have to make this job

Comment (vinyll) [reply circle.yml]: Maybe you just can't write into the directory that way and need to write into the [tmp dir](https://circleci.com/docs/2.0/configuration-reference/#example-for-root-key)?

However I'm not sure you still can access them afterwards, the docker will be turned down

Comment (vinyll) [reply circle.yml]: Proposal: Having a _dist_ or _prod_ github branch that hosts our files.

Comment (fuentesloic) [reply circle.yml]: and with this way Neomad could be call the prod one! right?

Comment (vinyll) [reply circle.yml]: considering a `dist` branch, neomad.org could import directly `https://www.gitcdn.xyz/repo/neomad-team/styleguide/dist/content/styles/main.css`

**Comment by fuentesloic** [comment]
@vinyll I push built file (only for styleguide.css) because for now, we need it to propulse on prod it.

When styleguide.neomad.org will do the job, we change it.

**Review by vinyll — APPROVED** [review]

**Pull request merged**